src/lib/commandclass/ColorSwitchCC.ts

```diff
@@ -93,6 +106,7 @@ const ColorComponentMap: Record<ColorKey, ColorComponent> = {
 	index: ColorComponent.Index,
 };
 const ColorKeys = keysOf(ColorComponentMap);
+const SupportedColorKeys = ColorKeys.map(supportedColorTableKey);
 function isColorKey(key: string): key is ColorKey {
 	// Note: indexing ColorComponentMap would be faster, but
 	//	the linter will not allow it.
```

Comment:
Do you have an example what you mean? I'm sure we can make it possible :)

Reply:
Previously, the code was something like
```js
function isColorKey(key: string): key is ColorKey {
	return ColorComponentMap[key as ColorKey] != undefined;
}
```
Weirdly, running the linter locally never complained, but the github lint check would.  I no longer remember what error it was giving me though.